### PR TITLE
Bump icon size

### PR DESCRIPTION
**Pull request by binford2k**

### CODE REVIEW

**Comment by binford2k** [comment]
screen shot 2016-03-28 at 1 45 37 pm

File: public/css/showoff.css [hunk 1]
```diff
@@ -763,8 +763,10 @@ body#download ul#downloads li {
    font-family: FontAwesome;
    font-style: normal;
    font-weight: normal;
+   font-size: 2em;
    text-decoration: inherit;
    padding: 0 8px;
+   float: left;
```
Comment (marrero984) [inline public/css/showoff.css:769]: why float?

**Comment by binford2k** [comment]
Because otherwise the line height looked funny. It's what makes the "drop cap" style work

File: public/css/showoff.css (hunk 1 above)
Comment (binford2k) [inline public/css/showoff.css:769]: Because otherwise the line height looked funny. It's what makes the "drop cap" style work

**Pull request merged**